tests/test_gt3x_loader.py

```diff
@@ -0,0 +1,16 @@
+"""Test the gt3x reader."""
+
+import pathlib
+
+from wristpy.core.models import Measurement, WatchData
+from wristpy.io.readers import readers
+
+
+def test_gt3x_loader() -> None:
```

Comment:
What happens if the input file is invalid e.g. you throw geneactiv into this loader, or a .txt file, or a non-existent file, or a file with the correct extension but corrupted data? 

Have a look at `with pytest.raises` and, if necessary, consider how to build a good try/catch around the actfast reader to catch any exceptions that it does not catch well.

Reply:
If a try/catch is necessary, ping me because it'll need some custom exception handling.

Reply:
I know that actfast handles corrupted/incomplete data. Will try to implement some error handling if we try to use the loaders for a file extension that is different from the explicitly stated one or a non-existent file